docs/tutorials/qpe_with_sampler.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>This is so much cleaner, thanks for taking a pass @Junye!!! I think the only things missing are just a little more setup about what phase estimation is (for folks that aren't already super familiar with Quantum) and how working this way with Sampler is different and hopefully easier than what the process looks like traditionally.</p><p> </p><p>Holler if any of the above doesn't make sense and thanks again</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Thanks for your review comments! I will work on those suggestions this week.</p>